metadata-load/src/test/java/uk/ac/ebi/ampt2d/metadata/importer/MetadataImporterMainApplicationAPITest.java

```diff
@@ -0,0 +1,72 @@
+/*
+ *
+ * Copyright 2019 EMBL - European Bioinformatics Institute
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+package uk.ac.ebi.ampt2d.metadata.importer;
+
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+import org.springframework.boot.SpringApplication;
+import org.springframework.context.ConfigurableApplicationContext;
+import uk.ac.ebi.ampt2d.metadata.persistence.repositories.StudyRepository;
+
+public class MetadataImporterMainApplicationAPITest {
```

Comment:
Could it be easier to use the Spring test runner class to avoid having to manually create the context and retrieve the repositories? Also applies to `MetadataImporterMainApplicationDBTest`.